app/javascript/packages/address-search/package.json

```diff
@@ -1,25 +1,29 @@
 {
   "name": "@18f/identity-address-search",
-  "private": true,
-  "version": "1.0.0",
+  "version": "1.0.3",
   "type": "module",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/18f/identity-idp.git",
-    "directory": "app/javascript/packages/address-search"
-  },
+  "private": false,
   "files": [
-    "index.tsx"
+    "dist"
   ],
+  "scripts": {
+    "prepublishOnly": "webpack"
+  },
+  "exports": {
+    ".": {
+      "source": "./index.tsx",
```

Comment:
Based on the failing build, I think we also need to make it so that Mocha tests resolve the `source` as well.

Looks like we can use the `--conditions` flag: https://nodejs.org/api/packages.html#resolving-user-conditions

Probably something we specify in https://github.com/18F/identity-idp/blob/main/.mocharc.js